packages/live-share-canvas/src/canvas/InkingCanvas.ts

```diff
@@ -253,12 +251,7 @@ export abstract class InkingCanvas {
     protected renderCircle(context: CanvasRenderingContext2D, center: IPoint, radius: number): void {
         const transformedCenter = this.viewportToScreen(center);
 
-        context.arc(
-            transformedCenter.x,
-            transformedCenter.y,
-            radius * this._scale,
-            0,
-            TWO_PI);
+        context.arc(transformedCenter.x, transformedCenter.y, radius * this._scale, 0, TWO_PI);
```

Comment:
That's not good. More than 3 parameters should be spread on individual lines.

Reply:
Added to re-vote